sdk/keyvault/keyvault-secrets/src/secretsModels.ts

```diff
@@ -46,9 +46,9 @@ export interface KeyVaultSecret {
  */
 export interface SecretProperties {
   /**
-   * @member {string} [vaultEndpoint] The vault URI.
+   * @member {string} [vaultUrl] The vault URI.
```

Comment:
I would suggest to keep the docs in sync with the `vaultUrl` passed to the client constructor. Unless this is a totally different value
```suggestion
   * @member {string} [vaultUrl] The base URL to the vault.
```